.travis.yml

```diff
@@ -28,10 +28,10 @@ before_install:
 
 install:
   - npm install -g testcafe
-  - cp local_travis.php local.php
-  - cp ./test/util/dbConfigTravis.json ./test/util/dbConfig.json
+  - cp .env.travis .env
   - composer install
   - ./travis-setup-db.sh
 
 script:
-  - testcafe chrome,firefox test/**/*.test.js --app "php -S 0.0.0.0:8888 -t ./"
\ No newline at end of file
+  - testcafe chrome,firefox test/**/*.test.js --app "php -S 0.0.0.0:8888 -t ./"
```

Comment:
is it possible to start the server and then use one of the npm scripts (`npm run test:e2e:XXX`) here?

Reply:
Good idea. Thanks!